cli.md

```diff
@@ -36,6 +36,8 @@ The `executable` could involve multiple command line tokens (e.g., `npm run` or
 - You cannot legally specify any argument more than one time
 - All codemod rules are loaded by default unless `--codemode-include` specifies a list.  `-codemode-exclude` works off all default codemods.
 - Specifying a `--codemod-include`or `--codemod-exclude`  that references a non-existent codemod will result in an error
+- The `--path-include` and `--path-exclude` patterns are interpreted as relative to the given project directory. In practice this means that the patterns should be joined with the project directory when used internally and also when passed to external tools.
+- Codemodders are free to determine reasonable defaults for included and excluded paths. In general, codemodders should attempt to match only relevant source files and to ignore test directories and build artifacts by default.
```

Comment:
I think it would be helpful to explain the default behavior and/or provide a link to a list of default excluded patterns, since that's actually the confusion for me. semgrep's exclude cli gives some helpful examples so maybe we could as well.

Codemodder by default will not analyze file patterns that match `" * */tests/ * *"` relative to the provided project directory. For example, running `codemodder src/tests ...` will analyze all files within `src/tests` except those in `src/tests/**tests/**

I'm not sure if that's helpful....